<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryPreferencesUI</class>
 <widget class="QWidget" name="HistoryPreferencesUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="chkLoadRecentMessages">
       <property name="text">
        <string>Load recent chat history in new chat windows</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Number of Messages to load:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinNoOfMessagesToLoad">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Database System:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cboDbSystem">
       <item>
        <property name="text">
         <string>SQLite</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MySQL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Postgre SQL</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>148</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chkLoadRecentMessages</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkLoadRecentMessages</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinNoOfMessagesToLoad</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
